lib/top/compat_top.ml

```diff
@@ -212,7 +138,7 @@ let add_directive ~name ~doc kind =
             in
             let id = Ident.create_persistent s in
             let sg = to_sig env loc id lid in
-            Printtyp.wrap_printing_env env (fun () ->
+            Printtyp.wrap_printing_env ~error:false env (fun () ->
```

Comment:
what's the reason for this new parameter?

Reply:
We had a `Printtyp` module which had a `wrap_printing_env` function which conditionally (dependent on OCaml version) added this parameter. Removing support for older version made this into a function that only added this parameter so I completely inlined it and removed our `Printtyp` compat shim.